config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,28 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password, :email, :name, :age, :employment_status, :primary_phone, :secondary_phone, :city, :state, :zip, :appointment_date, :insurance, :street_address_1, :street_address_2, :urgent_flag, :income, :special_circumstances, :procedure_cost, :procedure_date, :procedure_completed_date, :household_size, :full_text, :search]
+Rails.application.config.filter_parameters += [
```

Comment:
I have slight concerns that this can opt-in-list could drift over time 🤔 now that we're on full migrations & postgres, any opinion on adding [Danger](https://danger.systems/guides/getting_started.html) to yell at us anytime we add new columns, to think through if they get filtered here, added to archive patients, etc?

Reply:
I'm not EXTREMELY worried about this, but setting up `danger` or similar does feel like a pretty nice safety valve regardless. I'll spin up an issue for it - https://github.com/DARIAEngineering/dcaf_case_management/issues/2371